plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/panel/ProcessesPanelPresenter.java

```diff
@@ -99,8 +103,11 @@
                                                                       EnvironmentOutputEvent.Handler,
                                                                       DownloadWorkspaceOutputEvent.Handler {
 
-    public static final  String SSH_PORT              = "22";
-    private static final String DEFAULT_TERMINAL_NAME = "Terminal";
+    public static final  String SSH_PORT                    = "22";
+    private static final String DEFAULT_TERMINAL_NAME       = "Terminal";
+
+    public static final String TERMINAL_AGENT               = "org.eclipse.che.terminal";
```

Comment:
Can we move these variables to the shared constants? To avoid hardcoding them here.